CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ The types of changes are:
 ### Added
 * Added `FIDESOPS__DATABASE__ENABLED` and `FIDESOPS__REDIS__ENABLED` configuration variables to allow `fidesops` to run cleanly in a "stateless" mode without any database or redis cache integration
 
+### Changed
+* `MaskingStrategyFactory` and associated `MaskingStrategy` implementations now use a decorator-based registration system, to improve extensibility
```

Comment:
Given that there's no user-facing changes here, does this line actually belong under Developer Experience?

Reply:
yeah - i wasn't sure, i thought maybe dev experience was more strictly about tooling given the description that's listed? maybe this falls under "developer workflow"?

there's also a `Breaking Changes` tag which is specifically for public facing APIs, so i figured with that distinction, general back-end changes/maintenance probably still belongs under `Changed`. 

 but i don't have lots of experience with change logs so if you say it belongs there i'm happy to move it.

Reply:
Honestly this feels like a gray area. Maybe just leave it, and if someone else wants to move it, then that's their call 🤷‍♂️